### PR TITLE
DPC-860: Navigation bug on top tabs

### PR DESCRIPTION
**Pull request by Sun-Mountain**
**Why**

If you login and click on either the 'Support' or 'Documentation' tabs, you cannot navigate back to the 'Account' page unless you logout again. You can only get to the 'Profile' page. This is confusing to users.

**What Changed**

- Home button added to the navigation bar.
- When a user hovers over any of the three buttons, the color will change and the text will be underlined.
-  When a user hovers over DPC's title, the text changes color but does not have an underline to maintain the shape of the logo and text together.

Rails Update
- Updated rails version to `5.2.4` to fix javascript patch. [More info here](https://weblog.rubyonrails.org/2019/11/27/Rails-5-2-4-has-been-released/).

**Choices Made**

Screen Shot 2020-03-19 at 11 27 27 AM

Screen Shot 2020-03-19 at 11 27 29 AM

Screen Shot 2020-03-19 at 11 27 39 AM

The maximum size for the drop-down navigation bar was changed from `860px` to `962px` to accommodate the extra link and make sure all buttons are visible.
Screen Shot 2020-03-19 at 11 27 55 AM

Screen Shot 2020-03-19 at 11 40 02 AM

**Tickets closed**:

DPC-860

**Future Work**

[DPC-1207: Consolidate RoR App Stylesheets](https://jiraent.cms.gov/browse/DPC-1207)

**Checklist**

- [x] All tests are passing via `make ci-app` (app change) and `make ci-web` (website change)
- [x] Swagger documentation has been updated
- [x] FHIR documentation has been updated
- [x] Any required dpc-ops changes have a PR submitted and mentioned in this ticket
- [x] Any manual migration steps are documented, scripts written (where applicable), and tested
- [x] Before merging, any required dpc-ops changes have been approved and merged into master of the dpc-ops repo

### CODE REVIEW

File: dpc-web/app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -22,6 +22,12 @@ aria-controls="topnav-wrap" aria-label="Close menu">
       </svg>
     </button>
     <ul class="topnav">
+      <li class="topnav__item">
+        <%= link_to root_path, :class => "topnav__link" do %>
+          Home
+        <% end %>
```
Comment (switzersc-usds) [inline dpc-web/app/views/shared/_navbar.html.erb]: This can be shortened to:

```
<%= link_to "Home", root_path, class: "topnav__link" %>
```

Comment (switzersc-usds) [reply dpc-web/app/views/shared/_navbar.html.erb]: Ah, I just saw the other nav links are using the block pattern already. Not sure why -- it's longer and not necessary. I think the new link is fine as is for consistency, unless you feel like updating all the links to not use blocks. Up to you :)

File: dpc-web/app/assets/stylesheets/sidenav.scss [hunk 2]
```diff
@@ -1,3 +1,9 @@
+@import 'normalize.css/normalize';
+@import '@cmsgov/design-system-core/src/index';
+@import '@cmsgov/design-system-layout/src/index';
+@import "settings/font-faces";
+@import "settings/variables";
```
Comment (switzersc-usds) [inline dpc-web/app/assets/stylesheets/sidenav.scss]: How come this is necessary? These are all already being included in the app's stylesheets (app/assets/stylesheets/application.scss). I'm concerned about importing them again as 1) bad practice and 2) potentially loading all those files a second time.

Comment (dhgreene) [reply dpc-web/app/assets/stylesheets/sidenav.scss]: Thanks for making a [ticket](https://jiraent.cms.gov/browse/DPC-1207) for investigating this.

**Pull request merged**